modules/users/client/services/users.client.service.js

```diff
@@ -59,19 +63,104 @@
       },
       userSignin: function (credentials) {
         return this.signin(credentials).$promise;
-      }
+      },
+      getAllUnapprovedUsers: function () {
+        return this.getUnapprovedUsers().$promise;
+      },
+      approvalSwap: function () {
+        return this.approvalSwap().$promise;
+      },
+      remove: function () {
+        return this.remove().$promise;
+      },
     });
 
     return Users;
   }
 
+  angular
+    .module('users.admin.services')
+    .factory('ApplicantsService', ApplicantsService);
+
+  ApplicantsService.$inject = ['$resource'];
+
+  function ApplicantsService($resource) {
+    var Users = $resource('/api/users', {}, {
```

Comment:
You don't need all of these users resources since you are only dealing with applicants

Reply:
Yeah these are left over from Kevin's copypaste, I'll clean it up on the next commit.